UNIXFS.md

```diff
@@ -46,10 +46,15 @@ message Data {
 
 	optional uint64 hashType = 5;
 	optional uint64 fanout = 6;
+
+	repeated Metadata metadata = 7;
+	optional Metadata defaultMetadata = 8;
 }
 
 message Metadata {
-	optional string MimeType = 1;
+	optional string mimeType = 1;
+	optional uint32 mode = 2;
+	optional int64 mtime = 3;
```

Comment:
64 bits because:

1. We're using varints anyways.
2. I'd rather not deal with this again in 2038 even if we've moved on to a new format.

(also, it's signed because the world _did not_ start in 1970)

Reply:
> I'd rather not deal with this again in 2038 even if we've moved on to a new format.

❤️ this is some dedication to the project 😂

Reply:
Agree that the world didn't start in 1970 :) In that case should the default value be something other than 01/01/1970?